### ISSUE TITLE
Verification failed error 

### ISSUE THREAD

**Issue by mitsest** [r0adkll/sign-android-release#17]
Hello, I 'm getting verification failed error at the zipalign step.

It only works if I add extractNativeLibs="true" to the application tag in AndroidManifest.xml.

I 'm using gradle 3.6.3

Any hints?

**Comment by villeraikko** [comment]
I'm suddenly having this same issue with APK built with Ionic / Cordova.

**Comment by villeraikko** [comment]
I can also confirm that adding `android:extractNativeLibs="true"` to my AndroidManifest.xml fixes the problem, but this makes the generated APK much larger.

**Comment by HEYGUL** [comment]
Hi,
I have the exact same issue,  but adding android:extractNativeLibs="true" to my AndroidManifest.xml does not fix the problem.
Here is my workflow configuration file:
```
name: Android Release

on:
  push:
    tags:
      - '*'

jobs:
  test_build_and_distribute:

    runs-on: ubuntu-latest

    steps:
      - uses: actions/checkout@v2
        with:
          fetch-depth: 0
      - name: set up JDK 1.8
        uses: actions/setup-java@v1
        with:
          java-version: 1.8
      - name: Build with Gradle
        run: ./gradlew build

      - name: Unit Test
        run: ./gradlew testDebugUnitTest

      - name: Android Test Report
        uses: asadmansr/android-test-report-action@v1.2.0
        if: ${{ always() }} # IMPORTANT: run Android Test Report regardless

      - name: Build Release APK with Gradle
        run: ./gradlew assembleRelease

      - name: Sign Android release
        id: sign-android-app
        uses: r0adkll/sign-android-release@v1
        with:
          releaseDirectory: ./android/app/build/outputs/apk/release/
          signingKeyBase64: ${{ secrets.SIGNING_KEY }}
          alias: ${{ secrets.ALIAS }}
          keyStorePassword: ${{ secrets.KEY_STORE_PASSWORD }}
          keyPassword: ${{ secrets.KEY_PASSWORD }}

      - name: Upload artifact to Firebase App Distribution
        uses: wzieba/Firebase-Distribution-Github-Action@v1
        with:
          appId: ${{secrets.FIREBASE_APP_ID}}
          token: ${{secrets.FIREBASE_TOKEN}}
          releaseNotesFile: CHANGELOG.md
          groups: testers
          file: ${{ steps.sign-android-app.outputs.signedReleaseFile }}
```

Any idea ?

**Comment by thomastvedt** [comment]
> Hi,
> I have the exact same issue, but adding android:extractNativeLibs="true" to my AndroidManifest.xml does not fix the problem.

Same problem here... 🤔

**Comment by mitsest** [comment]
Hey all, from what  understood, newer versions of gradle are taking care of zipalign , so when this lib tries to zipalign again, the error is thrown.

I have managed to bypass this by cloning the repo and editing the typescript code to skip zipalign step.

**Comment by thomastvedt** [comment]
> Hey all, from what I understood, newer versions of gradle are taking care of zipalign , so when this lib tries to zipalign again, the error is thrown.
> 
> I have managed to bypass this by cloning the repo and editing the typescript code to skip zipalign step.
> 
> I 'll upload the code when I find some time and point you to it
> 
> Actually, someone already opened this as a PR: #24

You just beat me to it :)

**Comment by jobobby04** [comment]
If you want to use my action for now, you can just do use 
**jobobby04/sign-android-release@v1.0.2** 
or
**jobobby04/sign-android-release@v1.0.3**
1.0.3 is just dependencies being updated, and I havent tested it yet, 1.0.2 I can confirm works for newer android gradle plugins

**Comment by HEYGUL** [comment]
Thank you @jobobby04 ! It works like a charm.

**Comment by r0adkll** [comment]
Thanks, everyone! This should be resolved now as of #24
